plugins/corsair/corsair.quirk

```diff
@@ -0,0 +1,15 @@
+# SABRE RGB PRO WIRELESS
+[USB\VID_1B1C&PID_1B98]
+Plugin = corsair
+GType = FuCorsairDevice
+Name = CORSAIR SABRE RGB PRO WIRELESS Gaming Mouse
```

Comment:
The Name doesn't have the vendor, so remove `CORSAIR` please

Reply:
done